packages/trader/src/Modules/Account/Containers/account.jsx

```diff
@@ -26,11 +26,13 @@ class Account extends React.Component {
         WS.wait('authorize', 'get_account_status').then(() => {
             if (this.props.account_status) {
                 const { status } = this.props.account_status;
-                const is_high_risk_client = this.props.is_high_risk;
+                const needs_financial_assessment =
+                    this.props.is_high_risk || status.includes('financial_information_not_complete');
```

Comment:
We have this checks in `toggle-menu-drawer.jsx` which is needed for deriv.app responsive. Can you change there too?